spec/acceptance/type_spec.rb

```diff
@@ -1,29 +1,29 @@
 #! /usr/bin/env ruby -S rspec
 require 'spec_helper_acceptance'
 
-describe 'type function', :unless => (UNSUPPORTED_PLATFORMS.include?(fact('operatingsystem')) || is_future_parser_enabled?) do
+describe 'type function' do
```

Comment:
This test can be deleted completely if we don't support puppet3 anymore. See https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/lib/puppet/parser/functions/type.rb#L11